### PR TITLE
Update repo to match agreed DevOps Architecture

### PR DESCRIPTION
**Pull request by bethz**
Address part of issue #27 

Modify the repo to match the agreed DevOps Architecture.
Make Devops folder for yml.
Add folder devops with new yml. Removing current will be a future pr.
update devops name from interpret-extensions to interpret-community.

### CODE REVIEW

File: devops/nightly.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      include:
+        - master
+        - dev
+    always: true
+
+pool:
+  vmImage: "ubuntu-latest"
+
+variables:
+  artifactFeed: interpret-extensions-feed
```
Comment (wamartin-aml) [inline devops/nightly.yml]: What do the differences between these builds mean?

Comment (wamartin-aml) [reply devops/nightly.yml]: i.e. job vs steps

Comment (bethz) [reply devops/nightly.yml]: adding a job is first step to having multiple variants in test (Linux and windows and mac, then python 3.5, 3.6, 3.7) . Each of those will be a job and jobs can be named to reflect the variants while steps can not.

File: devops/code-coverage.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+        - dev
```
Comment (imatiach-msft) [inline devops/code-coverage.yml]: remove dev

File: devops/code-coverage.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    always: true
+
+pool:
+  vmImage: "ubuntu-latest"
+
+variables:
+  artifactFeed: interpret-extensions-feed
```
Comment (imatiach-msft) [inline devops/code-coverage.yml]: maybe we need to create a new feed, can be done in a later PR

File: devops/nightly.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+        - dev
```
Comment (imatiach-msft) [inline devops/nightly.yml]: remove dev here too

File: devops/code-coverage.yml (hunk 3 above)
Comment (bethz) [reply devops/code-coverage.yml]: true

File: devops/code-coverage.yml (hunk 2 above)
Comment (bethz) [reply devops/code-coverage.yml]: I removed code-coverage for now.

**Pull request merged**